pytest_pyodide/node_test_driver.js

```diff
@@ -55,7 +55,13 @@ rl.on("line", async function (line) {
     return;
   }
   if (line !== cur_uuid) {
-    cur_code += line + "\n";
+    line = line.substring(0,line.lastIndexOf('$'))
+    if( line === ""){
+      cur_code += "\n";
+    }else{
+      cur_code += line;
+    }
+    console.log("OK")
```

Comment:
If this is a delimiter, let's use a string that is a bit more clear and less likely to be passed by the user.

Also, please add some comment that cross-references the where this value is used.